test/test.html

```diff
@@ -1,4 +1,13 @@
-<!doctype html>
-<title>can-observable-mixin Tests</title>
-<script src="../node_modules/steal/steal.js" main="test/test"></script>
-<div id="qunit-fixture"></div>
+<!DOCTYPE html>
```

Comment:
if you're wondering why I do this everywhere https://stackoverflow.com/questions/55231157/live-reload-is-not-possible-without-body-or-head-tag ... (I also can't stand "partial" html pages)